src/Core/Checkout/Payment/Cart/PaymentTransactionChainProcessor.php

```diff
@@ -80,6 +80,13 @@ public function process(
         $criteria->addAssociation('transactions.paymentMethod');
         $criteria->addAssociation('lineItems');
 
+        $criteria->addAssociation('deliveries');
```

Comment:
Please rebase your PR, as the `addAssociationPath()` method is no longer there, you can use the `addAssociation()` method with an path instead.

Also it is not necessary to first add the `deliveries` or `address` association, when you later add nested associations from these.

```suggestion
        $criteria->addAssociation('deliveries.shippingMethod');
        $criteria->addAssociation('deliveries.shippingOrderAddress.country');
        $criteria->addAssociation('addresses.country');
        $criteria->addAssociation('addresses.salutation');

```

would be all you need.